beacon_chain/networking/eth2_network.nim

```diff
@@ -2395,18 +2431,34 @@ proc createEth2Node*(rng: ref HmacDrbgContext,
       anonymize = true,
       maxMessageSize = GOSSIP_MAX_SIZE_BELLATRIX,
       parameters = params)
+    
+    torpubsub = GossipSub.init(
+      switch = torSwitch,
+      msgIdProvider = msgIdProvider,
+      # We process messages in the validator, so we don't need data callbacks
+      triggerSelf = false,
+      sign = false,
+      verifySignature = false,
+      anonymize = true,
+      maxMessageSize = GOSSIP_MAX_SIZE_BELLATRIX,
```

Comment:
Nimbus doesn't use `GOSSIP_MAX_SIZE_BELLATRIX` anymore, so maybe this is an older commit? In the last few weeks, replaced all of this with just `GOSSIP_MAX_SIZE` (which now has the 10MB value `GOSSIP_MAX_SIZE_BELLATRIX did).

Reply:
The forked _experimental branch would be synced. The flag will be changed as wll